src/v0/destinations/zendesk/transform.js

```diff
@@ -285,8 +292,8 @@ async function getUserId(message, headers, type) {
   }
 }
 
-async function isUserAlreadyAssociated(userId, orgId, headers) {
-  const url = `${endPoint}/users/${userId}/organization_memberships.json`;
+async function isUserAlreadyAssociated(userId, orgId, headers, baseEndpoint) {
+  const url = `${baseEndpoint}/users/${userId}/organization_memberships.json`;
   const config = { headers };
   const response = await axios.get(url, config);
```

Comment:
Can we add a try-catch block here for now ? Otherwise we would get notified that something bad is happening in our source-code.

Also once scurry through the source-code to see for any unhandled(not surrounding with try-catch) axios/http calls

Reply:
updated